packages/grafana-eslint-rules/README.md

```diff
@@ -111,3 +111,34 @@ const getStyles = (theme: GrafanaTheme2) => ({
 ### `theme-token-usage`
 
 Used to find all instances of `theme` tokens being used in the codebase and emit the counts as metrics. Should **not** be used as an actual lint rule!
+
+### `no-untranslated-literals`
+
+Check if strings are marked for translation.
+
+Right now, we only check if a string is wrapped up by the `Trans` tag. We currently do not apply this rule to props, attributes or similars.
+
+```tsx
+// No untranslated strings ❌
+<InlineToast placement="top" referenceElement={buttonRef.current}>
+  Copied
+</InlineToast>
+
+// Wrap strings with `<Trans />` ✅
+<InlineToast placement="top" referenceElement={buttonRef.current}>
+  <Trans i18nKey="clipboard-button.inline-toast.success">Copied</Trans>
+</InlineToast>
+
```

Comment:
i think we could do with a `{t()}` example here:

```tsx
// Good ✅
<InlineToast placement="top" referenceElement={buttonRef.current}>
  {t('clipboard-button.inline-toast.success', 'Copied')}
</InlineToast>
```

Reply:
🤔 But we are not checking this. Would this not trigger misunderstandings?

Reply:
i still think we should have this as a `t()` example as well:
```tsx
// Good ✅
<InlineToast placement="top" referenceElement={buttonRef.current}>
  {t('clipboard-button.inline-toast.success', 'Copied')}
</InlineToast>
```

Reply:
🤔 But the internationalization doc says: 
`Prefer using <Trans /> for JSX children, and t() for props and other javascript usage.`
If we are this example, are we not giving the wrong example?